src/core/load-fragment.c

```diff
@@ -6645,3 +6646,27 @@ int config_parse_protect_hostname(
         free_and_replace(c->private_hostname, h);
         return 1;
 }
+
+int config_parse_bpf_delegate(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        uint64_t *sz = data;
+
+        assert(rvalue);
+
+        if (streq(rvalue, "any")) {
+                *sz = ~0ULL;
```

Comment:
UINT64_MAX

BTW, I think it is better to use `bpf_delegate_from_string()` here.